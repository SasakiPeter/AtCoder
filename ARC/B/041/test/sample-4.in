6 5
04030
40703
04030
00500
05050
00500
