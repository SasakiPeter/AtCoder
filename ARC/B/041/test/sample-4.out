00000
04030
00000
00000
00500
00000
